content/chainguard/chainguard-enforce/authentication/custom-idps/index.md

```diff
@@ -0,0 +1,194 @@
+---
+title : "Using Custom Identity Providers to Authenticate to Chainguard"
```

Comment:
I agree, I wasn't sure how to bucket this. I think I'll put the assumed identities doc in this same bucket, so that will hopefully make it feel more fleshed out